extensions/opencv/MotionDetector.cpp

```diff
@@ -201,6 +201,8 @@ void MotionDetector::onTrigger(const std::shared_ptr<core::ProcessContext> &cont
 void MotionDetector::notifyStop() {
 }
 
+REGISTER_RESOURCE(MotionDetector, "Detect motion from captured images."); // NOLINT
```

Comment:
why is the `// NOLINT` needed?

Reply:
as it was recently removed on main, it must be a rebase-artefact, deleted